src/python/ensembl/io/genomio/genome_stats/compare.py

```diff
@@ -189,11 +191,32 @@ def main() -> None:
     )
     parser.add_argument_src_path("--ncbi_stats", required=True, help="NCBI dataset JSON file")
     parser.add_argument_src_path("--core_stats", required=True, help="Core database JSON file")
+    parser.add_argument("-v", "--verbose", action="store_true", required=False,
+        help="Verbose level logging")
+    parser.add_argument("-d", "--debug", action="store_true", required=False,
+        help="Debug level logging")
     args = parser.parse_args()
 
+    # Configure logging
+    date_format='%Y/%m/%d_%I:%M:%S(%p)'
+    logging_format='%(asctime)s - %(levelname)s - %(message)s'
+    reload(logging)
+    log_level = None
+    if args.debug:
+        log_level = logging.DEBUG
+    elif args.verbose:
+        log_level = logging.INFO
+
+    logging.basicConfig(
+        format=logging_format, datefmt=date_format,
+        filemode="w", level=log_level
+        )
+
     try:
         ncbi_stats = get_json(args.ncbi_stats)["reports"][0]
+        logging.info(f"{args.ncbi_stats} JSON .'reports' obtained")
```

Comment:
This logging message should be in the `else` clause (in case you are not familiar with this, see second code block above [here](https://docs.python.org/3/tutorial/errors.html#raising-exceptions)).